contracts/masset/Masset.sol

```diff
@@ -729,21 +943,46 @@ contract Masset is
                     INFLATION
     ****************************************/
 
+    /**
+     * @dev Converts recently accrued swap fees into mAsset
+     * @return swapFeesGained        Equivalent amount of mAsset units that have been generated
+     * @return newSupply             New total mAsset supply
+     */
+    function collectInterest()
+        external
+        onlySavingsManager
+        nonReentrant
+        returns (uint256 swapFeesGained, uint256 newSupply)
+    {
+        uint256 toMint = 0;
+        if(surplus > 1){
+            toMint = surplus.sub(1);
+            surplus = 1;
```

Comment:
It might be good to add a comment here to explain why surplus isn't set to 0 (see https://mips.mstable.org/MIPS/mip-4). This seems to retain a tiny amount of fees in the contract. However, the "stuck" amount is negligible.